CHANGELOG.md

```diff
@@ -381,7 +381,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - 2012-12-10 Renamed Money::getUnits() to Money::getAmount()
 
 
-[Unreleased]: https://github.com/moneyphp/money/compare/v3.3.0...HEAD
```

Comment:
This should remain here and there should be an unreleased section in the beginning of the template.